docs/data-sources/cloud_private_data_source_connect_networks.md

```diff
@@ -0,0 +1,47 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_cloud_private_data_source_connect_networks Data Source - terraform-provider-grafana"
+subcategory: "Cloud"
+description: |-
+  Fetches Private data source connect networks from Grafana Cloud.
```

Comment:
```suggestion
  Fetches Private Data source Connect networks from Grafana Cloud.
```

Reply:
doh i see this is generated, but please update in the source

Reply:
Fixed in source